tox.ini

```diff
@@ -0,0 +1,52 @@
+[tox]
+envlist = py3, pycodestyle, pyflakes, pylint
+recreate = True
+
+[testenv]
+basepython = python3.6
+setenv =
+    FAKE_TRANSLATE = always
+deps =
+    urwid==1.2.1
+    PyYAML
+    attrs
+    systemd-python
+    jsonschema
+    pyudev
+    requests
+    requests-unixsocket
+    --editable=git+https://github.com/CanonicalLtd/probert@670c7258014eef13a971be0bf12206d96f2829f3#egg=probert
+
```

Comment:
I know you are working out issues with probert and systemd-python, but can we use requiremtnes.txt or test-requirements.txt to have these in one place?

Reply:
well... its kind of weird here.
in cloud-init we have 'requirements.txt' which are the runtime requirements and 'test-requirements.txt' that are *added* to those requirements to create the test environment.

this is different though, in the tox environment we specifically have a subset of the runtime requirements + the requirements needed for these tests. So re-using 'requirements' and 'test-requirements' doesn't really seem to fit for me.

lastly, one huge value of tox is the ability to pin versions, as we do in 'xenial-py3' things for cloud-init.  To create such an environment, you're going to have to list these requirements in tox.ini anyway... thus creating the duplication you were hoping to avoid.

Reply:
"we specifically have a subset of the runtime requirements + the requirements needed for these tests"

I guess I had hoped that test-requirements.txt would be used for that very thing.

"you're going to have to list these requirements in tox.ini anyway"

Is this something we assume we will use in the future or know we have to use now? Not that I want to kick the can down the road, but also want to be sure we aren't skipping using the test-requirements.txt file for something we may do.